### DISCUSSION TITLE
An unhandled exception occurred: Cannot find module 'webpack'

### DISCUSSION THREAD

**Discussion by Pawanupadhyay10** [polypheny/Polypheny-DB#321] [Q&A] [answered]
hey, help me out ,from this error ,i tried so many ways but it is not removing.
error:-
"An unhandled exception occurred: Cannot find module 'webpack'
Require stack:

    C:\Users\HP\Desktop\Polypheny-UI\node_modules@angular-devkit\build-webpack\src\webpack\index.js
    C:\Users\HP\Desktop\Polypheny-UI\node_modules@angular-devkit\build-webpack\src\index.js
    C:\Users\HP\Desktop\Polypheny-UI\node_modules@angular-devkit\build-angular\src\dev-server\index.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\node_modules@angular-devkit\architect\node\node-modules-architect-host.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\node_modules@angular-devkit\architect\node\index.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\models\architect-command.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\commands\serve-impl.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\node_modules@angular-devkit\schematics\tools\export-ref.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\node_modules@angular-devkit\schematics\tools\index.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\utilities\json-schema.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\models\command-runner.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\lib\cli\index.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\lib\init.js
    C:\Users\HP\AppData\Roaming\npm\node_modules@angular\cli\bin\ng
    See "C:\Users\HP\AppData\Local\Temp\ng-MK2YJn\angular-errors.log" for further details."

**Comment by datomo** [comment, marked as answer]
Hey @Pawanupadhyay10 
Thank you for your question. 
Could you provide some more information how this error ocurred?
What part of Polypheny did you try to install/setup/start?
Which command did you use?
